.gitignore

```diff
@@ -6,3 +6,6 @@ cache/
 log/
 *.swp
 .DS_Store
+
```

Comment:
Please, remove the changes in `.gitignore`, there are no ide-specific folders there!

Reply:
Done